lib/common/core/CmdParser.cpp

```diff
@@ -591,10 +591,11 @@ int CmdLineArgsParser::Parse(__in LPWSTR oneArg) throw()
                 PrintUsage();
                 return -1;
             }
-            else
+            else if ('-' == CurChar())
```

Comment:
This would also allow `/-es6all` for example to parse correctly. Is that expected?